src/assert.sh

```diff
@@ -20,7 +20,7 @@ function assert_equals_ignore_colors() {
   local label="${3:-$(helper::normalize_test_function_name "${FUNCNAME[1]}")}"
 
   local actual_without_colors
-  actual_without_colors=$(echo -e "$actual" | sed "s/\x1B\[[0-9;]*[JKmsu]//g")
```

Comment:
This change was needed because of `tput sgr0`:
https://stackoverflow.com/a/35582778